admin/sql/create_types.sql

```diff
@@ -1,3 +1,4 @@
 CREATE TYPE eval_job_status AS ENUM ('pending', 'running', 'done', 'failed');
 CREATE TYPE model_status AS ENUM ('hidden', 'evaluation', 'show');
 CREATE TYPE version_type AS ENUM ('lowlevel', 'highlevel');
+CREATE TYPE id_type AS ENUM ('mbid', 'mesbid');
```

Comment:
We have standardised on `msid` as the abbreviation for messybrainz ids.

You can see in the github view of this commit that there is an indicator at the end of this line saying "no newline". It's a generally accepted "rule" to always have one newline at the end of a file.